projects/js-packages/connection/components/use-connection/index.jsx

```diff
@@ -77,6 +77,16 @@ export default ( {
 		} );
 	};
 
+	const handleRegisterSiteOnly = e => {
```

Comment:
Alternatively, if you're keeping this method, I'd recommend updating [the tests](https://github.com/Automattic/jetpack/blob/trunk/projects/js-packages/connection/components/use-connection/test/use-connection.jsx) for it as well